src/mappings/event/loans.ts

```diff
@@ -100,17 +102,17 @@ export async function newMarket(
         token: currency,
         height: height,
         timestamp: timestamp,
-        borrowCap: market.borrowCap,
-        supplyCap: market.supplyCap,
+        borrowCap: divideByTenToTheNth(market.borrowCap, decimals),
+        supplyCap: divideByTenToTheNth(market.supplyCap, decimals),
         rateModel: InterestRateModel,
-        closeFactor: market.closeFactor,
+        closeFactor: market.closeFactor / 10**6,
```

Comment:
Two things:
1. Please use `divideByTenToTheNth` for these values for consistency, and
2. Please use a constant instead of magic numbers, and a comment or name that indicates why 10**6 (instead of any other number).

Reply:
`divideByTenToTheNth is only needed if the input is a big number.
A number can be divided safely by 10**6. Using the function would make it more complicated.
The factor was arbitrarily chosen by your designers, it's a convention. If you want some name what should it be? DECIMALS_FOR_RATIO perhaps? Where should be defined?

Reply:
@daniel-savu , do you have a suggestion for the name? I don't know what the origin is for the decimal factor of 10**6

Reply:
It's `10 ** 6` because the number is represented as units per million (`Permill` parachain type). The `interbtc-api` lib already has a decoding function for this, including the constant naming (`PERMILL_BASE`). It would be great if you could import and use that function: https://github.com/interlay/interbtc-api/blob/a629d9a846804cf207838794f368606c7ebc7a10/src/utils/encoding.ts#L378